src/Form/Input.js

```diff
@@ -17,27 +17,31 @@ const InputContainer = createComponent({
 const StyledInput = createComponent({
   name: 'Input',
   tag: 'input',
-  style: ({ isFloating, size, theme, borderRadius = theme.radius }) => css`
+  style: ({ isFloating, size, theme, borderRadius = theme.radius, leftIcon, rightIcon }) => css`
     border: 1px solid ${theme.colors.greyLight};
-    height: ${theme.heights[size]}px;
+    height: ${theme.inputHeights[size]}px;
     display: block;
     outline: none;
     width: 100%;
-    padding: 8px;
+    padding: 8px 24px 8px 8px;
     border-radius: ${borderRadius}px;
     transition: 250ms all;
     -webkit-appearance: none;
     font-family: inherit;
     font-size: ${theme.fontSizes[size]}px;
+    color: ${theme.colors.greyDarkest};
 
     &:hover,
-    &:focus,
     &:active {
       border-color: ${theme.colors.greyDark};
     }
 
+    &:focus {
+      border-color: ${theme.colors.primary};
+    }
+
     ::placeholder {
-      color: ${theme.colors.greyDark};
+      color: ${theme.colors.grey};
     }
 
     &[disabled] {
```

Comment:
The disabled UI in FF is inconsistent with Chrome/Safari. Can we look into making FF have a similar look as other browsers? 
<img width="856" alt="Screen Shot 2019-08-19 at 10 25 52 AM" src="https://user-images.githubusercontent.com/6404663/63286012-e6bdd300-c26b-11e9-858b-32c765b4507d.png">